controllers/status.go

```diff
@@ -86,13 +86,19 @@ func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGenerat
 	})
 }
 
-// sets conditions when volume is demoted and requeued for resync
-func setOnlyDegradedCondition(conditions *[]metav1.Condition, observedGeneration int64) {
+// sets conditions when volume is demoted and requeued for resync without errors
```

Comment:
sets conditions when volume is demoted, resync is completed and volume is ready to use?